Example/Tests/TransactionLog/EthTransactionLogTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+// This source file is part of the Web3Swift.io open source project
+// Copyright 2019 The Web3Swift Authors
+// Licensed under Apache License v2.0
+//
+// EthTransactionLogTests.swift
+//
+// Created by Vadim Koleoshkin on 24/05/2019
+//
+
+import Nimble
+import Quick
+@testable import Web3Swift
+
+class EthTransactionLogTests: XCTestCase {
+    
+    func testLogFetching() {
+        expect{
+            try EthTransactionHash(
+                transactionHash: BytesFromHexString(
+                    hex: "0xfbaad83ca594fa30902586f06c9a2fd4d3855f3769e4cf3157eb0013abd0661f"
+                ),
+                network: MainnetAlchemyNetwork()
+            ).receipt().logs().value()[0]
+        }.notTo(
+            throwError(),
+            description: "Expect log to be obtained"
+        )
+    }
+    
+    func testLogFetchingFields() {
```

Comment:
could you pls fix indentation across this test and remove extra line breaks in the end